### DISCUSSION TITLE
modules for functions

### DISCUSSION THREAD

**Discussion by kuivi** [CliDyn/climsight#40] [Ideas]
**Motivation:** Currently, we have one extensive file that can only be executed using _Streamlit_. This setup is inconvenient for performing tests on functions. 

**Here's my proposal:**
Create 2 to 3 additional files, specifically Python modules. These modules will encapsulate all the functions responsible for retrieving information, organized into logical categories (e.g., geography, physics, law, plotting, etc.).

The main file, climsight.py, will import these modules. Technically, this will not alter the process for running the application: streamlit run climsight.py.

This approach allows for an additional file dedicated to executing all the functions and performing tests to ensure their functionality without the need for Streamlit.

**Comment by koldunovn** [comment]
What you describe is moving to "normal" python package structure. Let's address it in #44 :)
